src/Middleware/RequestThrottling/test/MiddlewareTests.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.SignalR.Tests;
```

Comment:
Change the namespace of `SyncPoint`

Reply:
Should I make a new namespace? Maybe `Microsoft.AspNetCore.SyncPoint`?

Reply:
I would use `Microsoft.AspNetCore.Internal`